steps/olm/operator.go

```diff
@@ -160,3 +160,43 @@ var _ = gauge.Step("Verify <resourceType> Results records", func(resourceType st
 var _ = gauge.Step("Verify <resourceType> Results logs", func(resourceType string) {
 	operator.VerifyResultsLogs(resourceType)
 })
+
+var _ = gauge.Step("Verify <resourceType> signature", func(resourceType string) {
+	operator.VerifySignature(resourceType)
+})
+
+var _ = gauge.Step("Start the kaniko-chains task", func() {
+	operator.StartKanikoTask()
+})
+
+var _ = gauge.Step("Verify image signature", func() {
+	operator.VerifyImageSignature()
+})
+
+var _ = gauge.Step("Check attestation", func() {
+	operator.CheckAttestation()
+})
+
+var _ = gauge.Step("Verify attestation", func() {
+	operator.VerifyAttestation()
+})
+
+var _ = gauge.Step("Verify that image registry variable is exported", func() {
+	if os.Getenv("CHAINS_REPOSITORY") == "" {
+		testsuit.T.Errorf("'CHAINS_REPOSITORY' environment variable is not exported")
+	}
+})
+
+var _ = gauge.Step("Create signing-secrets for Tekton Chains", func() {
+	if oc.SecretExists("signing-secrets", "openshift-pipelines") {
+		log.Printf("Secrets \"signing-secrets\" already exists")
+		operator.CreateFileWithCosignPubKey()
```

Comment:
IMHO this should go in `else` branch of the following `if ` statement. At this point the secret might exist but might be empty